### ISSUE TITLE
TODO before release

### ISSUE THREAD

**Issue by Makin-Things** [Makin-Things/platinum-weather-card#1]
This is a list of things that need to be done before release (mostly so Simon doesn't forget stuff). There are 3 sections, the first list are the essential things, below that is a list of nice to have, followed by a bunch of ideas/maybe/optional (to get things moved up the list please open a separate issue for discussion)

### Essential
- [x] Rename card to platium-weather-card
- [x] Add option to use attribute of entity for extended forecasts
- [x] Add tooltips to the horizontal forecasts
- [x] Add the current days extended forecast section
- [x] Figure out how to report the card size correctly
- [x] Make sure the chrome 102 entity picker bugs are resolved
- [x] Massively update documentation
- [x] Add daily forecast config option for extended entity when using vertical
- [x] Add daily forecast config option to allow selecting how many days to show the extended text for
- [x] For vertical forecast, abort adding days as soon as an unknown day is found (as an example using the bom integration you get 7 days between about 4pm and 12am, the rest of the time it is only 6)
- [x] Ensure everything works ok on a mobile device
- [x] Human friendly names for slots (need help/ideas)
- [x] Add slot values as per comment comment below (need a name for these, both internal and human friendly)
- [x] Allow allowing the sections to be reordered
- [x] Allow the daily forecast to use attributes for the extended forecast
- [x] Write code to clean up config
- [x] Remove testing debug messages
- [x] Add styling pages
- [x] Make sure all existing options are added to the editor or removed from the card (the list below has the options the still need adding, comment if something is missing and not in this list)
  - [x] Pressure decimals
  - [x] Today decimals for temps
  - [x] Old daily format
  - [x] Temp format
- [x] Standardize config option names (this will be fun as it has the potential to break things)
- [x] Clean up the alt config rubbish
- [x] Fix calculation of day in forecast sections
- [x] Add options for overview section (default 'combined', but add options to show only observations or forecast info)
- [x] Replace icons with new set
- [x] Add 2 more slot rows (ie. L7,L8,R7,R8)

### Under consideration
- Consider what, if any css customization we include in the config option (trying to avoid these where possible to avoid bloat in the config editor)
- Consider more flexibility in what is displayed in daily forecasts (as suggested in https://community.home-assistant.io/t/custom-dark-sky-animated-weather-card/59816/1369)
- ~~While editing the card, highlight the section being edited by showing a red border around the section. I am thinking that while on the main config page, hovering over the option block on the left would highlight on the card and once in a subsection leave the section being worked on highlighted.~~ The HA devs tell me this isn't possible right now.

**Comment by DavidFW1960** [comment]
Add slots for min/max with actual and forecast values like
'''
Todays Min 12o (Fore 9o)
'''

Doing above on iphone so looks like crap. 

Add 2 more rows of slots

**Comment by DavidFW1960** [comment]
> Add 2 more rows of slots

![image](https://user-images.githubusercontent.com/36390762/173280272-433bc98a-3527-4e2e-8aeb-2d9aa4ca18b4.png)
So here are some more weather entities I include on a different card
![image](https://user-images.githubusercontent.com/36390762/173280410-00eabbef-aab9-42e5-9526-3a3e963c08b6.png)
Also illuminance.
2 more rows of slots would let me add a few of these to the main card. I think that's a valid use case.

**Comment by Makin-Things** [comment]
Closing as 1.0.0 has been released.
